client/state/selectors/get-editor-url.js

```diff
@@ -17,10 +19,23 @@ export const getEditorUrl = ( state, siteId, postId = null, postType = 'post' )
 			url = `${ siteAdminUrl }post.php?post=${ postId }&action=edit`;
 		}
 
+		if ( 'gutenberg-redirect-and-style' === getSelectedEditor( state, siteId ) ) {
```

Comment:
Testing with an Atomic site, the selected editor is `classic` meaning this condition is false.

Reply:
should we consider the _classic editor forcerd_ condition instead of?

```es6
if ( ! isClassicEditorForced( state, siteId ) ) {
  // ...
}
```

Reply:
You're right! It turns out that we really only need to check to for VIP here, as the rest of the logic in `isClassicEditorForced` is already handled with the use of `isEligibleForGutenframe`. I've updated the logic in fd7b3d7

Reply:
Nice Let's mark it as resolved.